CMakeLists.txt

```diff
@@ -38,19 +38,16 @@ CHECK_MINIMAL_CXX_STANDARD(11 ENFORCE)
 ADD_PROJECT_DEPENDENCY(dynamic-graph REQUIRED PKG_CONFIG_REQUIRES dynamic-graph)
 ADD_PROJECT_DEPENDENCY(pinocchio REQUIRED PKG_CONFIG_REQUIRES pinocchio)
 ADD_PROJECT_DEPENDENCY(example-robot-data)
-SET(BOOST_COMPONENTS filesystem system thread program_options
-  unit_test_framework regex)
+#ADD_PROJECT_DEPENDENCY(Boost REQUIRED COMPONENTS filesystem system thread program_options unit_test_framework regex)
```

Comment:
Is it commented out on purpose ?

Reply:
We don't need all of that, I just wanted to check but forgot what I was doing… I'm fixing it.